nsdb-cluster/src/main/resources/application-common.conf

```diff
@@ -26,8 +26,12 @@ akka {
   actor {
     provider = cluster
 
+    allow-java-serialization = on
```

Comment:
to point out that this is temporary (until a different method for serialization will be implemented in the library) you could add a comment or a fixme